packages/widgets/src/widgets/SealModuleWidget/components/PositionSummary.tsx

```
@@ -331,22 +331,22 @@ export const PositionSummary = () => {
           hasPositions &&
           isUpdatedValue(existingRewardContract?.toLowerCase(), selectedRewardContract?.toLowerCase()) ? (
             [
-              isRewardContractTokensLoading || !existingRewardContractTokens ? (
+              isRewardContractTokensLoading ? (
                 <Skeleton className="w-30 h-5" />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)